src/main/resources/liveTemplates/ATG.xml

```diff
@@ -0,0 +1,15 @@
+<templateSet group="ATG">
+    <template name="trytd" value="try (AutoCloseTransactionDemarcation td =&#10;             new AutoCloseTransactionDemarcation($SELECTION$$TRANSACTION_MANAGER$$END$, TransactionDemarcation.REQUIRED)) {&#10;    // TODO: your code goes here&#10;    td.setSuccess(true);&#10;} catch (TransactionDemarcationException e) {&#10;    throw e;&#10;}" description="Auto-closed transaction template" toReformat="true" toShortenFQNames="true">
+        <variable name="TRANSACTION_MANAGER" expression="" defaultValue="&quot;transactionManager&quot;" alwaysStopAt="true" />
```

Comment:
such templates require manual imports
use fqdn for auto-imports - https://intellij-support.jetbrains.com/hc/en-us/community/posts/206961285-type-auto-import-on-live-templates